clang/test/CodeGenSYCL/loop_unroll.cpp

```diff
@@ -1,7 +1,7 @@
-// RUN: %clang_cc1 -triple spir64-unknown-unknown-sycldevice -disable-llvm-passes -fsycl-is-device -emit-llvm %s -o - | FileCheck %s
-
+// RUN: %clang_cc1 -triple spir64-unknown-unknown-sycldevice -disable-llvm-passes -fsycl-is-device -sycl-std=2020 -emit-llvm %s -o - | FileCheck %s
```

Comment:
@bader could you please take a look at the changes in this test to confirm behavior is as required. This test was incorrect since -fsycl wasn't passed earlier. The test started failing once all required options were set. If the test isn't behaving as expected, please file a bug report. I don't think we need to hold up this PR since fixing the test (if it needs to be) is outside the scope of this patch

Reply:
Changes looks okay to me, but it was added by @vmaksimo, so I would rather get her feedback instead.

I think setting `-sycl-std=2020` should be unnecessary as loop unroll metadata generation must be invariant to SYCL version.

Reply:
Looks okay, I agree with @bader

Reply:
I removed the `-sycl-std=2020` from the test, thanks for checking!